common/remote/rmtsmtp.cpp

```diff
@@ -472,8 +472,10 @@ class CMailInfo
     unsigned port;
     StringAttr sender;
     Owned<ISocket> socket;
+    Owned<ISecureSocket> secureSocket;
```

Comment:
Is this used ?

Reply:
Good catch. That was part of the test change and If forgot to remove it.